src/ProjectService.Models.Db/Entities/DbRole.cs

```diff
@@ -0,0 +1,14 @@
+﻿using System;
+using System.ComponentModel.DataAnnotations;
+
+namespace LT.DigitalOffice.ProjectService.Models.Db.Entities
+{
+    public class DbRole
+    {
+        [Key]
+        public Guid Id { get; set; }
```

Comment:
Fluent API can be used. It is necessary to clarify about this case with @DarkFelas . He at the last call said that they are preferable, because they have a higher priority. But should you try to use it every time?

Reply:
These changes are from the develop branch